test/CMakeLists.txt

```diff
@@ -25,11 +26,15 @@ foreach(program ${API_TESTS})
     "${CMAKE_BINARY_DIR}/bin/${program}")
   set_tests_properties("Test${program}" PROPERTIES ENVIRONMENT
     LSAN_OPTIONS=suppressions=${CMAKE_CURRENT_SOURCE_DIR}/data/asan.supp)
+  if(HERMES_ENABLE_COVERAGE)
+    set_coverage_flags(${program})
```

Comment:
I don't think we need the coverage flags on each test. We're only interested in coverage in the library, not which lines get executed in the tests themselves. Right?

Reply:
I have seen both cases, including test or not. Either is fine.